package.json

```diff
@@ -3,13 +3,14 @@
   "publishConfig": {
     "registry": "https://npm.pkg.github.com/"
   },
-  "version": "3.0.0",
+  "version": "3.0.1",
   "description": "parametrize search queries",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "test": "yarn tsc",
-    "prepublish": "yarn tsc"
+    "build": "yarn tsc",
+    "build:watch": "yarn tsc -w",
```

Comment:
you could just `yarn build -w`, that's even shorter ;)

Reply:
Sure `yarn build -w` works.. but the idea is that if someone does not know about the `-w` option of `tsc` he can check it out in `package.json -> scripts`